lib/datadog/appsec/extensions.rb

```diff
@@ -112,6 +114,14 @@ def trace_rate_limit
           @settings.trace_rate_limit
         end
 
+        def obfuscator_key_regex
+          @settings.obfuscator_key_regex
+        end
+
+        def obfuscator_value_regex
+          @settings.obfuscator_key_regex
+        end
```

Comment:
Wait, is this change (and a few others on this file) a new feature? It didn't quite look like this existed before.

Reply:
It was actually an overlook, which typing pointed me to! So it's a consistency fix for an existing feature.